protocol/protobuf/application_metadata_message.proto

```diff
@@ -99,5 +99,6 @@ message ApplicationMetadataMessage {
     COMMUNITY_PUBLIC_SHARD_INFO = 79;
     SYNC_COLLECTIBLE_PREFERENCES = 80;
     COMMUNITY_USER_KICKED = 81;
+    DELETE_ALL_MEMBER_MESSAGES = 82;
```

Comment:
This will remove all existing msgs on the client till a certain date.
The corner case is messages, that we are fetching as they will not be removed, but I'm not 100% sure that we need to handle them.

Reply:
What if sending a signal from the memberId of a contact that has been banned from the community and deleting messages on the client regardless of backend?